### PR TITLE
feat(meshloadbalancingstrategy): fix affinity tags and percentage

### PR DESCRIPTION
**Pull request by slonka**
xrel: https://github.com/kumahq/kuma-website/issues/1653

waiting on kuma implem

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_src/policies/meshloadbalancingstrategy.md [hunk 1]
```diff
@@ -448,9 +448,9 @@ spec:
         localityAwareness:
           localZone:
             affinityTags:
-              - key: infra.io/datacenter
+              - key: kubernetes.io/hostname
```
Comment (slonka) [inline app/_src/policies/meshloadbalancingstrategy.md:451]: this is still missing in this PR https://github.com/kumahq/kuma/pull/9690/files - @lukidzi - any reason for not including it?

Comment (lukidzi) [reply app/_src/policies/meshloadbalancingstrategy.md:451]: Based on https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#node-labels I noticed only zone and region

Comment (slonka) [reply app/_src/policies/meshloadbalancingstrategy.md:451]: PR added https://github.com/kumahq/kuma/pull/10243

Comment (lukidzi) [reply app/_src/policies/meshloadbalancingstrategy.md:451]: We need to change all examples. I think there is more examples with tags

Comment (slonka) [reply app/_src/policies/meshloadbalancingstrategy.md:451]: there is only this https://github.com/kumahq/kuma-website/blob/c6b449fdd7331518893e8449a4c30fde52e9108c/app/_src/policies/meshloadbalancingstrategy.md#L348-L349

**Review by michaelbeaumont — APPROVED** [review]

**Pull request merged**